src/main/java/com/rackspace/ceres/app/model/Metric.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2020 Rackspace US, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.rackspace.ceres.app.model;
+
+import java.time.Instant;
+import java.util.Map;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+import lombok.Data;
+
+@Data
+public class Metric {
+  @NotNull
+  Instant timestamp;
+  @NotBlank
+  String metric;
+  @NotEmpty
+  Map<String,String> tags;
+  @NotNull
+  Number value;
```

Comment:
Does this mean string/boolean metrics are being dropped?

Reply:
For now, yes. This is a PoC.